src/cli.rs

```diff
@@ -109,9 +111,20 @@ impl Cli {
     pub fn compile_args(&self) -> CompileArgs {
         CompileArgs {
             debug: self.debug,
-            verbosity: self.hugr_args.other_args.verbose.log_level(),
+            verbosity: self.verbose.log_level(),
             validate: self.validate,
             qsystem_pass: self.qsystem_pass,
         }
     }
+
+    fn wrap_generator(generator: Option<String>, val_err: PackageValidationError) -> CliError {
```

Comment:
This should be a helper method in `CliError`. I added it in https://github.com/CQCL/hugr/pull/2507.


```suggestion
    // TODO: Replace with `CliError::validation` in `hugr-cli >= 0.22.2`.
    fn wrap_generator(generator: Option<String>, val_err: PackageValidationError) -> CliError {
```